data_architect_misc/data_transformer/transform_functions/latam_el_salvador_transform_functions.py

```diff
@@ -0,0 +1,54 @@
+"""This is the subclass of Transform function for El Salvador (LATAM division).
+
+We will define transform functions specific to El Salvador here.
+
+Author: Maicol Contreras
+Last Modified: August 3, 2020
+"""
+
+import pandas as pd
+
+from constants import comp_harm_constants
+from transform_functions.common_comp_harm_transform_functions import CommonCompHarmTransformFunctions
+
+
+class LatamElSalvadorTransformFunctions(CommonCompHarmTransformFunctions):
+    """
+    All custom (uncommon) transform functions **SPECIFIC to
+    individual processing task** must be defined as part
+    of this class.
+    """
+    EL_SALVADOR_SPECIFIC_CATEGORY_MAPPINGS = {
+        "(?i).*DETERGENTES/JABONES.*ROPA.*": "Home Care",
```

Comment:
It seems like some of these mappings are the same between El  Salvador, Nicaragua and Honduras. Why don't we try to put them in comp_harm_constants.CATEGORY_MAPPINGS? If you think these are Spanish and therefore, they should be separated from English ones, feel free to create SPANISH_CATEGORY_MAPPINGS in comp_harm_constants.py. Then rename the 'CATEGORY_MAPPINGS' to 'ENGLISH_CATEGORY_MAPPINGS' in comp_harm_constants.py. Finally, you do this in comp_harm_constants.py:
CATEGORY_MAPPINGS = dict(SPANISH_CATEGORY_MAPPINGS, **ENGLISH_CATEGORY_MAPPINGS)

and then in the 'LatamElSalvadorTransformFunctions', you can combine them like this:
 dict(comp_harm_constants.CATEGORY_MAPPINGS,
                                 **LatamElSalvadorTransformFunctions.EL_SALVADOR_SPECIFIC_CATEGORY_MAPPINGS)

Of course, you'll leave the El Salvador specific mappings in the 'latam_el_salvador_transform_functions.py' file. 

The goal here is not never repeat the same stuff in different code files. Otherwise, we'll forget to update them all when we find a bug or something. Hope that makes sense. Let me know if you have questions.